src/client/pages/Offer/Introduction/ExternalInsuranceProvider/index.tsx

```diff
@@ -4,7 +4,7 @@ import {
   useExternalInsuranceDataQuery,
   useExternalInsuranceDataStatusSubscription,
 } from 'data/graphql'
-import { OfferData } from 'pages/OfferNew/types'
+import { OfferData } from 'src/client/pages/Offer/types'
```

Comment:
I guess `src/client` should be removed here right? :) The auto import messes up the paths, a bit annoying :D 
```suggestion
import { OfferData } from 'pages/Offer/types'
```